### PR TITLE
Layout login

### LINKED ISSUE #18: Layout login

### PR DESCRIPTION
**Pull request by Roandersonxd1**
Fix #18

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -37,7 +37,11 @@ dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation 'androidx.core:core-ktx:1.3.2'
+
```
Comment (leonardobrito) [inline android/app/build.gradle:40]: Precisa dessas linhas em branco?

File: android/app/src/main/res/drawable/ic_back.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (leonardobrito) [inline android/app/src/main/res/drawable/ic_back.xml:1]: Ainda precisa desse arquivo?

File: android/app/src/main/res/layout/activity_login.xml [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+<!--        <ImageView-->
+<!--            android:layout_width="wrap_content"-->
+<!--            android:layout_height="wrap_content"-->
+<!--            android:background="@drawable/ic_back"-->
+<!--            android:padding="5dp">-->
+<!--        </ImageView>-->
```
Comment (leonardobrito) [inline android/app/src/main/res/layout/activity_login.xml:22]:
```suggestion
```

File: android/app/src/main/res/layout/activity_login.xml [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+            android:text="@string/create_account"
+            android:textColor="@color/black"
+            android:textAllCaps="false"
+            android:layout_gravity="center"
+            android:background="@android:color/transparent">
+        </Button>
+
+    </LinearLayout>
+
+</ScrollView>
```
Comment (edimossilva) [inline android/app/src/main/res/layout/activity_login.xml:152]: Add empty line

File: android/app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,3 +1,12 @@
...
+
+    <!--Screen Login-->
+    <string name="login_tile">Login</string>
+    <string name="enter_username_hint">Digite seu email</string>
+    <string name="enter_password_hint">Digite sua senha</string>
+    <string name="rememberme">Lembrar senha</string>
+    <string name="forget_password">Esqueceu sua senha ?</string>
+    <string name="create_account">Criar conta</string>
+
 </resources>
```
Comment (edimossilva) [inline android/app/src/main/res/values/strings.xml:12]: Add empty line

File: android/app/src/main/java/com/example/myos/ui/LoginActivity.kt [hunk 6]
```diff
@@ -0,0 +1,12 @@
+package com.example.myos.ui
```
Comment (edimossilva) [inline android/app/src/main/java/com/example/myos/ui/LoginActivity.kt:1]: I think you should rename this package

**Review by edimossilva — APPROVED** [review]
I think you should 

- delete some extra "empty spaces"
- add  an empty line to the end of some files
- rename the "example" package

**Pull request merged**